articles/build-a-restful-api-with-adonisjs/index.md

```diff
@@ -0,0 +1,419 @@
+### Build A RESTful API With AdonisJS
+
+### Outline
+Introduction
+Setting up AdonisJS
+Creating Databases
+Setting up Authorization and Authentication
+Creating Models
+Creating Controllers
+Creating Endpoint Routes
+Testing the Forum API
+Conclusion
+
+### Introduction
+In today’s ever-changing and rapid world of frameworks including frontend and backend frameworks, learning and building RESTful APIs to bridge the gap between clients and data is a demanding skill for any software engineer.
```

Comment:
I would say `In today’s ever-changing and rapid world of frontend and backend frameworks,`
Rather than
`In today’s ever-changing and rapid world of frameworks including frontend and backend frameworks,`

Reply:
done